lib/bitcoin/BitcoinClient.ts

```diff
@@ -5,6 +5,8 @@ import BitcoinInputModel from './models/BitcoinInputModel';
 import BitcoinLockTransactionModel from './models/BitcoinLockTransactionModel';
 import BitcoinOutputModel from './models/BitcoinOutputModel';
 import BitcoinTransactionModel from './models/BitcoinTransactionModel';
+import BitcoinWallet from './BitcoinWallet';

```

Comment:
someday we won't be bundling bitcoin dependencies like this... sigh...

Reply:
:) I think we are ready for the mono repo change @OR13

Reply:
don't joke about such things :) ... I am waiting for @thehenrytsai and @csuwildcat to tell me we are good to move to that structure...